Automattic/jetpack
# Jetpack: Masterbar: Return correct type for get_available_languages filter

## Proposed changes:

Code in Masterbar hooks the get_available_languages filter on WoA sites to return an empty value to disable a language dropdown. `__return_null` was used to do this, which works but can make other code that tries to actually use the value warn (in PHP <8) or fail (in 8.0+) since it's expecting an array.

Change it to `__return_empty_array` which will return the correct type.

### Other information:


## Jetpack product discussion


Probably the cause of p9F6qB-bmA-p2

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* I think this needs a WoA dev site running with PHP 8, as the relevant code is guarded with a `$this->site_woa` check. Try reproducing the issue described in p9F6qB-bmA-p2, then apply this change and see if it fixes it.
